spec/lib/i18n_interpolation_fallbacks_spec.rb

```diff
@@ -32,32 +32,32 @@
       before do
         @old_locale = I18n.locale
         I18n.locale = 'it'
-        I18n.backend.store_translations('it', {"nonexistant_key" => "%{random_key} here is some Italian"})
+        I18n.backend.store_translations('it', {"nonexistent_key" => "%{random_key} here is some Italian"})
```

Comment:
- Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.
- Redundant curly braces around a hash parameter.
- Prefer annotated tokens (like `%<foo>s`) over template tokens (like `%{foo}`).